Library/include/CSP/Multiplayer/SpaceEntity.h

```diff
@@ -123,6 +123,9 @@ class CSP_API SpaceEntity
 	// General callback providing success/fail boolean.
 	typedef std::function<void(bool)> CallbackHandler;
 
+	// Callback that will provide a pointer to a SpaceEntity object.
+	typedef std::function<void(SpaceEntity*)> EntityCreatedCallback;
```

Comment:
When should the developer expect to bind to this callback?

Reply:
This had to be duplicated from SpaceEntitySystem so the wrapper generator didnt break. This is for CreateChildEntity